addons/point_of_sale/static/src/app/screens/product_screen/order_summary/order_summary.js

```diff
@@ -350,4 +351,95 @@ export class OrderSummary extends Component {
         }
         return newLine;
     }
+    get isComboApplicable() {
+        return this.pos.applicableProductCombo.length > 0;
+    }
+    get bestPotentialCombos() {
+        const bestCombos = {
+            applicable: [],
+            upsell: [],
+        };
+        this.pos.applicableProductCombo.forEach((applicable) => {
+            applicable.comboPrice = applicable.productTmpl.getPrice(
+                this.currentOrder.pricelist_id,
+                applicable.combinations.length
+            );
+            applicable.numberOfUpsell = Object.values(applicable.combinations[0]).reduce(
+                (acc, combo) => {
+                    if (combo.upsell) {
+                        return acc + 1;
+                    }
+                    return acc;
+                },
+                0
+            );
```

Comment:
You're editing return value of a getter inside another getter that's discouraged.

Getter should never reassign values.